CONTRIBUTING.md

```diff
@@ -1,8 +1,6 @@
 # Developer Guide
 
-# TODO (andreyvelich): This doc needs to be updated for Kubeflow Trainer V2
-
-Kubeflow Training Operator is currently at v1.
+# TODO (Trainer V2 Update):  The testing and local Docker image building instructions need to be updated for Kubeflow Trainer V2. The current instructions in the repository are based on the Training Operator V1 and may not be directly applicable to the Trainer V2 project..
 
 ## Requirements
```

Comment:
Let's update the Kind version from here: https://github.com/kubeflow/trainer/blob/master/go.mod